src/components/copyListOfStories/script.js

```diff
@@ -9,17 +10,28 @@
 
 	function _init() {
 		_scope = document.querySelector('.IterationTracking_DetailTracking_Grid, .TeamRoom_ListView');
+		if (_scope) {
+
+			if (!document.querySelector('link[href*="copyListOfStories/styles.css"]')) {
+				injectStyle(_options.componentsUrl + 'copyListOfStories/styles.css', _appendTeamsCopyButtons);
+			} else {
+				_appendTeamsCopyButtons();
+			}
+			return
+		}
 
-		if (!_scope) { return; }
-
-		if (!document.querySelector('link[href*="copyListOfStories/styles.css"]')) {
-			injectStyle(_options.componentsUrl + 'copyListOfStories/styles.css', _appendCopyButtons);
-		} else {
-			_appendCopyButtons();
+		_scope = document.querySelector('.WorkitemPlanning_TaskTestList_Grid');
+		if (_scope) {
```

Comment:
looks messy.
how about just add new selector to first _scope declaration. And further just check that _scope has workitemplanning class. Actually you can make new variable which will save needed callback by condition.
Like that:
`var appendButtons = _scope.classList. contains('WorkitemPlanning_TaskTestList_Grid') ? _appendBacklogCopyButtons : _appendTeamsCopyButtons`